### ISSUE TITLE
Dropbox sign in form opens outside installed webapp

### ISSUE THREAD

**Issue by Bubka** [keeweb/keeweb#1085]
**Describe the bug**
Dropbox sign in fails when Keeweb is used as a web app on iOS because the Dropbox form pops up in Safari.

**To Reproduce**
1. Open Keeweb web app in Safari for iOS
2. Add the app to the home screen (via the Safari share button)
3. Open Keeweb app for the spingboard
4. Click `+` then the Dropbox icon to set your app key then th Dropbox icon again to sign in

=> Safari starts and displays the Dropbox form

**Expected behavior**
The Dropbox sign in form should pop up directly in the web app instead of launching Safari.

Seems to be a regression, see: #298 Dropbox popup doesn't work in installed webapp

**Environment**
iOS 12.1 on iPhone XS

**Comment by antelle** [comment]
Interesting, looks like it changes in new iOS version.

**Comment by antelle** [comment]
I don't see any way to solve this. All methods I tried open external links in Safari instead of the webapp.

**Comment by antelle** [comment]
If anyone knows a way to open a link in iOS Safari Web Applications, please let me know, I'll add it to KeeWeb. Testing is simple: create a webapp and try to navigate to http://example.com inside it.

**Comment by Bubka** [comment]
arf, bad news.

But how do you explains that v1.6.3 still works using the same device?! Something may have change in v1.7.x code that fire this behaviour ?!

**Comment by antelle** [comment]
wow interesting, does it open Dropbox right there in the app? or you’re already authorized?

**Comment by Bubka** [comment]
Yep, right there in the app, just after a fresh install of v1.6.3 on my springboard.

**Comment by antelle** [comment]
nice, I’ll check this today, probably that’s because of manifest or something else, but if it works, it’s possible to fix :)

**Comment by antelle** [comment]
So, it happens because of manifest.json. If we remove it, everything works as before. I haven't found any way to fix it and keep manifest.json.  
While this fixes the thing, it sucks hard. Removal of manifest.json is blocking service workers and probably breaks icons in Android. In the same time, deprecated appcache will eventually stop working, probably in a year or so. Not sure how to fix it in a sustainable way, but for now we'll be able to enjoy Dropbox again once it's released.

**Comment by antelle** [comment]
Wow! 🔥🔥🔥
- Service workers are removed from the experiments
- External sites are loaded via SFViewController
- The current state of any app is maintained

This is a huge improvement.
